source/support/rotate-secrets/index.html.md.erb

```diff
@@ -0,0 +1,168 @@
+used by an app, and are set by its deployment task, for example the
+[GOV.UK Notify API key][].
+
+Secrets are referenced in the Concourse configuration with `((secret
+name))` so, to take the daily statistics task as an example, here
+`report-bearer-token` is a secret made available to the task through
+the `BEARER_TOKEN` environment variable:
+
+```yaml
+- task: trigger-bigquery-export
+  file: git-main/concourse/tasks/trigger-bigquery-export.yml
+  params:
+    CDN_DOMAIN: account.publishing.service.gov.uk
+    BEARER_TOKEN: ((report-bearer-token))
+```
+
+You can list all secrets for the GOV.UK Concourse team with the
+gds-cli:
+
+```
+$ gds cd secrets ls cd-govuk-tools
+```
+
+[the daily statistics task]: https://github.com/alphagov/govuk-account-manager-prototype/blob/7e69b8cb8daec1e6e23d32374df3f8a1fa45b477/concourse/pipeline.yml#L249-L271
+[GOV.UK Notify API key]: https://github.com/alphagov/govuk-account-manager-prototype/blob/5443585c3d2dd5d399efc18f126f0bdf831036b5/concourse/tasks/deploy-to-govuk-paas.yml#L59
+
+### Rotating a secret
+
+To rotate a secret:
+
+1. Generate a new secret value, which varies per secret.
+
+2. Set the new value with the gds-cli:
+
+    ```
+    $ gds cd secrets add cd-govuk-tools <pipeline-name>/<secret-name> "<value>"
+    ```
+
+3. If this is a secret used by an app, deploy the app.
+
+## How to rotate the secrets we have
+
+In general secrets can be rotated by generating a new value and
+setting the value with the gds-cli.  Those few secrets which need more
+care are explained further.
+
+### Secrets used by multiple pipelines
+
+#### `paas-username` and `paas-password`
+
+The password can be changed by [triggering a password reset][], which gets sent to the govuk-accounts-developers Google Group.
+
+[triggering a password reset]: https://admin.london.cloud.service.gov.uk/password/request-reset
+
+#### `sentry-dsn`
+
+Generate a new DSN from the Sentry project settings page.
+
+#### `slack_webhook_url`
+
+Inform IT Support that our Slack webhook has been leaked, and that we
+need a new one.
+
+### Secrets used by the account manager
+
+#### `basic-auth-username` and `basic-auth-password`
+
+These are arbitrary strings.  After changing them, update the
+"deployed applications" Trello card.
+
+#### `bigquery-credentials-production`
+
+#### `notify-api-key-production` and `notify-api-key-staging`
+
+Generate a new API key from [GOV.UK Notify][].
+
+[GOV.UK Notify]: https://www.notifications.service.gov.uk/
+
+#### `oidc-pepper-production` and `oidc-pepper-staging`
+
+Generate an arbitrary string with `rake secret`.
+
+This secret is used to [generate subject identifiers][], and so
+rotating it will cause services to see a new subject identifier for a
+returning user.  This means that services won't be able to identify
+returning users, and will treat them as new users.
+
+If this secret needs to be rotated, we will need to engage with other
```

Comment:
This is a really good point and one we should cover in our "team docs" vs "gov.uk" docs chat tomorrow.
If we were going to have a "how to rotate the Brexit checker secrets" doc that would sit well in the GOV.UK developer docs.

I guess when we integrate more services we'll need some kind of process for handing this with lots of different support functions 🤔 or we'd break a ton of apps overnight...